vite.config.lite.js

```diff
@@ -0,0 +1,29 @@
+import { resolve } from 'path'
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+  build: {
+    sourcemap: true,
+    outDir: 'dist-lite',
+    lib: {
+      // Could also be a dictionary or array of multiple entry points
+      entry: resolve(__dirname, 'src/index-lite.js'),
+      name: 'ScrollTimelineLite',
+      // the proper extensions will be added
+      fileName: (format, entryAlias) => `scroll-timeline-lite${format=='iife'?'':'-' + format}.js`,
+      formats: ['iife'],
+    },
+    minify: 'terser',
+    terserOptions: {
+      keep_classnames: /^((View|Scroll)Timeline)|CSS.*$/
+    },
+    rollupOptions: {
+      output: {
+        // Provide global variables to use in the UMD build
+        // for externalized deps
+        globals: {
+        },
+      },
+    }
+  },
+})
```

Comment:
This is duplicating a lot of config that likely should be kept in sync. Can you move all of this to a file like `vite.common.js`, e.g.
```js
export function buildConfig(source, filename) {
    return {
        build: {
            sourcemap: true,
            emptyOutDir: false,
            lib: {
            // Could also be a dictionary or array of multiple entry points
            entry: source,
            name: 'ScrollTimeline',
            // the proper extensions will be added
            fileName: (format, entryAlias) => `${filename}${format=='iife'?'':'-' + format}.js`,
            formats: ['iife'],
            },
            minify: 'terser',
            terserOptions: {
            keep_classnames: /^((View|Scroll)Timeline)|CSS.*$/
            },
            rollupOptions: {
            output: {
                // Provide global variables to use in the UMD build
                // for externalized deps
                globals: {
                },
            },
            }
        }
    };
}
```

vite.config.js:
```js
import { resolve } from 'path'
import { defineConfig } from 'vite'
import { buildConfig } from './vite.common'

export default defineConfig(buildConfig(resolve(__dirname, 'src/index.js'), 'scroll-timeline'));
```

vite.config.lite.js:
```js
import { resolve } from 'path'
import { defineConfig } from 'vite'
import { buildConfig } from './vite.common'

export default defineConfig(buildConfig(resolve(__dirname, 'src/index-lite.js'), 'scroll-timeline-lite'));
```

This also eliminates the need to have a separate `dist` dir by configuring vite to not empty the dir on building.

Reply:
Done.